tfhe/src/core_crypto/algorithms/misc.rs

```diff
@@ -218,6 +218,7 @@ mod test {
         for _ in 0..NB_TESTS {
             let num: f64 = rng.gen();
             let mut denom = 0.0f64;
+            #[allow(clippy::while_float)]
```

Comment:
what's this ?

Reply:
A float comparison in a loop
https://rust-lang.github.io/rust-clippy/master/#/while_float

Reply:
weird lint given how we use it